libs/langchain/langchain/callbacks/manager.py

```diff
@@ -1634,6 +1709,37 @@ def configure(
         )
 
 
+class AsyncChainGroupCallbackManager(AsyncCallbackManager):
```

Comment:
Wrapping like this seems like the easiest way to not break backwards compat while also avoiding conditional return values in the context manager, though it introduces yet another callback manager class